grammars/rescript.tmLanguage.json

```diff
@@ -127,7 +127,7 @@
         {
           "name": "string.template",
           "begin": "([a-z_][0-9a-zA-Z_]*)?(`)",
-          "end": "`",
+          "end": "[^\\]`",
```

Comment:
Should this be ```"[^\\\\]`"``` to json encode the escaped backslash.

Reply:
why so many backslash? `\\` is already an escaped backslash, isn't it?

Reply:
or is it because we need to add another layer of escaping because it's a json? Then yes!

Reply:
yeah I guess this makes sense:
```suggestion
          "end": "[^\\\\]`",
```